### ISSUE TITLE
Custom Titles in pages

### ISSUE THREAD

**Issue by Seluj78** [OpenApprenticeFoundation/openapprentice-website#46]
Have custom titles used and passed when rendering template like done in https://github.com/OpenApprenticeFoundation/openapprentice-blog/blob/afe59afbc1ec5121de94e82540fc8dfbc1d4e205/openapprentice_blog/templates/basic_layout.html#L8-L12

**Comment by davidkartuzinski** [comment]
@Seluj78 - This means that one the templates, there needs to be a title entry section for each page. So as I create pages of content, or when we do a blog post, we enter the title and then this code will take it. Or else it will just not change anything.

**Comment by Seluj78** [comment]
Yeah well for the blog (cc @shenoyprath123), We will specify the title of the post, it will be stored in the DB and displayed as page title etc.

For the website, I was thinking that we could just have, like we already have, a entry in the render template:
```python
render_template("template.html", title="xxx")
```
